controllers/user.js

```diff
@@ -39,6 +39,34 @@ async function add(req, res) {
   });
 }
 
+function getShareByEmail(req) {
+
+  const subject = encodeURIComponent(req.fluentFormat("share-by-email-subject"));
+  const body = encodeURIComponent(req.fluentFormat("share-by-email-message"));
+
+  return {
+    "gmail" : {
+      client: "Gmail",
```

Comment:
Are we sure these email client names don't need translation?

Reply:
Pretty sure, but to be totally sure...
@flodolo should Gmail, Yahoo, and Outlook be translated?

Reply:
I'd say no, but can you give me the context in which they are used?

Reply:
@flodolo They appear in a list of email clients a user can select to open an email in. Currently, only "Email" and "Other" get translated.
<img width="324" alt="screen shot 2018-11-29 at 9 16 24 am" src="https://user-images.githubusercontent.com/22355127/49231422-83c39d80-f3b7-11e8-8dce-d56e54bf7d6b.png">

Reply:
OK, then I would say to keep them not translatable (I was afraid they would appear in the middle of a sentence), and see if anyone complains about it.